src/app/shared/pipes/reason-transform.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'reasonTransform'
```

Comment:
bad naming, this pipe can be reused for different purposes, so it is better to rename it according to its functionality